config/envs/default.py

```diff
@@ -30,3 +30,57 @@ class DefaultConfig(object):
     RSA256_PUBLIC_KEY_BASE64 = getenv("RSA256_PUBLIC_KEY_BASE64")
     if RSA256_PUBLIC_KEY_BASE64:
         RSA256_PUBLIC_KEY: str = base64.b64decode(RSA256_PUBLIC_KEY_BASE64).decode()
+
+    # Content Security Policy
+    SECURE_CSP = {
+        "default-src": "'self'",
+        "script-src": [
+            "'self'",
+            "'sha256-GUQ5ad8JK5KmEWmROf3LZd9ge94daqNvd8xy9YS1iDw='",
+            "'sha256-0F02u9outG+s4SWeWHc/Pmk8iJNh7GtqrTtLPzkxZ3M='",
```

Comment:
This is something that hasn't been done well in pre-award, but that I'd recommend: can you add a comment next to each line indicating what script this is allowing? Makes it a bit more auditable and understandable 🙏  I'm assuming it's the two scripts you've converted from `<script>...</script>` to `<script src=""/>`

Reply:
If it _is_ this files, consider using CSP nonces instead of these script hashes - should be a bit more maintainable as the hashes won't need updating if the file contents change:

https://github.com/communitiesuk/funding-service-design-post-award-data-store/blob/a01b416375eef42cb70bb7dd13b856581f178dce/app.py#L158

and 

<img width="847" alt="image" src="https://github.com/user-attachments/assets/c1d4aee2-b1cf-4762-98fe-0178be091b79" />